src/payment/PaymentForm.test.jsx

```diff
@@ -18,16 +18,20 @@ const storeMocks = {
 configureI18n(configuration, messages);
 
 describe('<PaymentForm />', () => {
+  let wrapper;
+  let paymentForm;
+  beforeEach(() => {
+    wrapper = mount((
```

Comment:
Note: I think this could have been `let wrapper = ...` because I think it was only used here, but I could be wrong.

Reply:
Changed to `const wrapper`